deploy.php

```diff
@@ -0,0 +1,459 @@
+<?php
+
+namespace Deployer;
+
+use Symfony\Component\Yaml\Yaml;
+
+require 'recipe/symfony3.php';
+require 'recipe/cachetool.php';
+require 'recipe/sentry.php';
+
+// Define some variables
+set('client', '$client');
+set('project', '$project');
+set('repository', '$repository');
+set('release', 'current');
+set('production_url', '$productionUrl');
+set('sentry_organization', '$sentryOrganization');
+set('sentry_project_slug', '$sentryProjectSlug');
+set('sentry_token', '$sentryToken');
+
+// Define staging
+host('dev03.sumocoders.eu')
+    ->user('sites')
+    ->stage('staging')
+    ->set('deploy_path', '~/apps/{{client}}/{{project}}')
+    ->set('branch', 'staging')
+    ->set('bin/php', 'php7.4')
+    ->set('bin/composer', '{{bin/php}} /home/sites/apps/{{client}}/{{project}}/shared/composer.phar')
+    ->set('cachetool', '/var/run/php_74_fpm_sites.sock')
+    ->set('document_root', '~/php74/{{client}}/{{project}}');
+
+// Define production
+//host('$host')
+//    ->user('sites')
+//    ->stage('production')
+//    ->set('deploy_path', '~/apps/{{client}}/{{project}}')
+//    ->set('branch', 'master')
+//    ->set('bin/php', '$phpBinary')
+//    ->set('cachetool', '$socketPath')
+//    ->set('document_root', '~/php74/{{client}}/{{project}}');
+
+/*************************
+ * No need to edit below *
+ *************************/
+
+set('use_relative_symlink', false);
+
+// Shared files/dirs between deploys
+add('shared_files', ['.env.local', 'app/config/parameters.yml']);
+add('shared_dirs', ['src/Frontend/Files']);
+
+// Writable dirs by web server
+add('writable_dirs', ['src/Frontend/Cache', 'src/Backend/Cache']);
```

Comment:
Moet `src/Frontend/Files` daar ook niet bij? Of is shared map altijd schrijfbaar?

Reply:
@davysumo ?

Reply:
Je hebt gelijk, folders die toegevoegd worden aan shared_dirs zijn niet by default schrijfbaar.

Aangepast in https://github.com/sumocoders/forkcms/pull/249/commits/16764718347567a8ce02eabf2791293c6a8ffdd9